crates/fluvio-spu/src/services/public/stream_fetch.rs

```diff
@@ -185,6 +186,10 @@ impl StreamFetchHandler {
                     Ok(())
                 }
                 StreamFetchError::Socket(err) => Err(err),
+                StreamFetchError::Compression(err) => {
+                    error!(%err, "compression error");
```

Comment:
is there why this doesn't return err like others?

Reply:
right, I need to create a new variant in Error code for this

Reply:
I create a new error code and I am sending that when this error happens